docs/curriculum-topics-resources/open-source-licenses.md

```diff
@@ -4,75 +4,80 @@ sidebar_position: 2
-For software licenses, there are a number of acceptable licenses depending on your use case, including the GNU General Public license, the MIT license, or the BSD license. 
+For software licenses, there are a number of acceptable licenses depending on your use case, including the GNU General Public license, the MIT license, or the BSD license.
 
 ![](https://miro.medium.com/max/1400/0*Nbx3Uw8UgXQgHyzI.png)
 
-_**A licensee of BSD-licensed software can:**_
+## Hardware Licenses
+Open-source hardware refers to tangible objects such as machinery, mobile technology and other devices whose designs and other documentation for how to build the hardware are licensed under an open source license. Open source hardware designs, documentation, and bill of materials are available in a public repository so that they can be modified, changed or distributed.
+
+For hardware licenses, we recommend the **CERN, MIT, or TAPR Open Hardware Licenses**.
 
-* Use, copy and distribute the unmodified source or binary forms of the licensed program 
-* Use, copy and distribute modified source or binary forms of the licensed program provided that: 
-    * all distributed copies are accompanied by the license 
-    * the names of the previous contributors are not used to promote any modified versions without their written consent 
+For now, we will simply cover the CERN Open Hardware License, or OHL, developed by the CERN European Organization for Nuclear Research. It is to be included alongside the hardware design documentation files.
 
-We recommend this license because a licensee can take BSD-licensed code and add to it, safe in the knowledge that whatever they contribute **can be distributed in whatever way they choose.** For this reason the license is seen as friendly to traditional software business models that depend upon keeping the source private and capitalising on the sale of licensed binaries. 
+The [CERN OHL](https://ohwr.org/project/cernohl/wikis/home) allows anyone to see, study, modify, and share the design documentation for the piece of hardware to which it applies.  [See here for examples](https://ohwr.org/projects/cernohl/wiki/cernohlprojects).
 
-## Hardware Licenses
-Open-source hardware refers to tangible objects such as machinery, mobile technology and other devices whose designs and other documentation for how to build the hardware are licensed under an open source license. Open source hardware designs, documentation, and bill of materials are available in a public repository so that they can be modified, changed or distributed.  
+If modifications are made and distributed, it must be under the same license conditions – this is the ‘persistent’ nature of the license, which ensures that the whole community will continue benefiting from improvements, in the sense that everyone will in turn be able to make modifications to these improvements.
 
-For hardware licenses, we recommend the **CERN, MIT, or TAPR Open Hardware Licenses**. 
+## Content Licenses
```

Comment:
Added a content and data section, this is a topic I've come across a lot lately.